app/assets/stylesheets/components/about.scss

```diff
@@ -0,0 +1,198 @@
+#about {
+
+    .row {
+        padding: 40px;
+        color: $text-primary;
+    }
+    
+    .intro {
+        text-align: center;
```

Comment:
I actually intended all the text in this section to be center-aligned, there is no h2 tag here.